### DISCUSSION TITLE
When I try to save a large file the pipe breaks

### DISCUSSION THREAD

**Discussion by derdilla** [neutralinojs/neutralinojs#1022] [Q&A]
I am trying to save a medium sized (50mb) text file using `filesystem.writeFile`, the browser breaks:

`CONSOLE NETWORK ERROR WebSocket connection to 'ws://localhost:33677/' failed: Error sending data: Broken pipe`

Until a full application restart I am not able to save any files at all. 
Is this a issue or is this due to a common mistake?

**Comment by kedodrill** [comment]
Related issue: https://github.com/neutralinojs/neutralinojs/issues/1100
